test-utils/state-viewer/src/commands.rs

```diff
@@ -52,10 +49,22 @@ pub(crate) fn state(home_dir: &Path, near_config: NearConfig, store: Arc<Store>)
 
 pub(crate) fn dump_state(
     height: Option<BlockHeight>,
+    stream: bool,
+    mut single_file: bool,
+    file: Option<PathBuf>,
     home_dir: &Path,
     near_config: NearConfig,
     store: Arc<Store>,
 ) {
+    assert!(
+        !stream || !single_file,
+        "At most one of options --stream and --single_file can be set"
```

Comment:
yeah, you def shouldn't be writing code for this, clap can do that: 
Might use `mark::hit!();` here: 

https://github.com/rust-analyzer/rust-analyzer/blob/d54e1157b66017e4aae38328cd213286e39ca130/crates/test_utils/src/mark.rshttps://docs.rs/clap/latest/clap/struct.Arg.html#method.conflicts_with

(not super obvious how to plug that into derive, but should be possilbe)